assets/scss/_variables.scss

```
@@ -1,6 +1,6 @@
 //site color variables
-$body-color: #fff;
-$primary-color: #EAA4A4;
+$body-color: rgb(247, 241, 241);
+$primary-color: #f187ce;
```

This may be from the Hugo workshop, can you change this back to the original values?

```suggestion
$body-color: #fff;
$primary-color: #EAA4A4;
```